Makefile

```diff
@@ -332,6 +349,8 @@ precommit: prerequisites
 	$(Q)+$(MAKE) --no-print-directory clean
 	$(Q) echo "Running checks..."
 	$(Q)+$(MAKE) --no-print-directory check-signed-off check-vera check-cppcheck
+	$(Q) echo "...testing build-options..."
+	$(Q)+$(MAKE) --no-print-directory test-buildoptions
```

Comment:
With this version the runtime is more than twice before.
But now when the option-testing running in parallel and the build-option step included into the precommit process, the runtime only increased to ~5:55 minutes from ~3:25 on my local machine. So basically option-testing takes ~2:30 minutes.
What do you think, is this slow-down acceptable or not?